requirements-dev-frozen.txt

```diff
@@ -178,7 +178,7 @@ sentry-forked-djangorestframework-stubs==3.14.5.post1
 sentry-kafka-schemas==0.1.38
 sentry-redis-tools==0.1.7
 sentry-relay==0.8.39
-sentry-sdk==1.38.0
+sentry-sdk==1.39.0
```

Comment:
this won't do anything due to the override: https://github.com/getsentry/sentry/blob/cd6c02dc14d9449bb9e37059dee9b6d455bd6d58/requirements-getsentry-overrides.txt#L10

Reply:
I see, thanks!